lib/common.sh

```diff
@@ -108,6 +108,13 @@ if [[ ${os_version} -ne 7 ]] && [[ ${os_version} -ne 8 ]] && [[ ${os_version} -n
   exit 1
 fi
 
+# Use firewalld on CentOS/RHEL, iptables everywhere else
+export USE_FIREWALLD=False
+if [[ ($OS == "rhel" || $OS = "centos") && ${os_version} == 8 ]]
```

Comment:
nit, we could do this logic in ansible since it's not used anywhere in the scripts now?